src/org/scada_lts/dao/ViewDAO.java

```diff
@@ -182,6 +214,15 @@
 				+ COLUMN_NAME_MVU_VIEW_ID+"=? and "
 				+ COLUMN_NAME_MVU_USER_ID+"=?";
 
+	private static final String WATCHLIST_USERS_PROFILES_SELECT_BASE_ON_USERS_PROFILE_ID = ""
+			+ "select "
+			+ COLUMN_NAME_UP_VIEW_ID+ ", "
+			+ COLUMN_NAME_UP_PERMISSION + " "
+			+ "from "
+			+ "viewusersprofiles "
```

Comment:
viewusersprofiles can generate exceptions.
Try to use `viewUsersProfiles` literal.